src/http/request.ts

```diff
@@ -72,15 +72,24 @@ export class DrashRequest extends Request {
    * @param pathParams - The path params to match the request's URL to. The path
    * params come from a resource's path(s).
    * @param connInfo - The connection info Deno provides on a new request
+   * @param requestOptions - Any options to control the way requests behave.
    *
    * @returns A Drash request object.
    */
   static async create(
     request: Request,
     pathParams: Map<string, string>,
     connInfo: ConnInfo,
+    requestOptions?: RequestOptions,
   ) {
     const req = new DrashRequest(request, pathParams, connInfo);
+
+    if (requestOptions) {
+      if (requestOptions.read_body === false) {
```

Comment:
`ignore_body` could be a good candidate... when i first read `read_body`, i thought it was a required bool while `ignore_body` conveys that it's an optional setting 🤔

Reply:
i think this makes sense, but would we also make this change from the user's point of view? like would they pass in `ignore_body` when they create the server object?

Reply:
yeah, don't they have to do that now with `read_body`? currently, `read_body` defaults to `true` right? so i can see `ignore_body` defaults to `false`... but honestly no strong feelings ha